paystack/src/main/java/co/paystack/android/utils/CardUtils.java

```diff
@@ -50,6 +50,19 @@ public static boolean isNotExpired(int year, int month) {
     return !hasYearPassed(year) && !hasMonthPassed(year, month);
   }
 
+  public static String retrieveLast4Digits(String number){
```

Comment:
I understand that this logic was just moved here. Do you think there should be error handling on input and maybe throwing an exception instead of returning the input value?